public/api-documentation/swagger.json

```diff
@@ -401,6 +441,32 @@
         }
       }
     },
+    "SubmitSignature": {
+      "type": "object",
+      "properties": {
+        "answer": {
+          "type": "string"
+        },
+        "question_id": {
+          "type": "string"
+        },
+        "token": {
+          "type": "string"
+        },
+        "user_id": {
+          "description": "CDX user id",
+          "type": "string"
+        }
+      }
+    },
+    "SubmitTransaction": {
```

Comment:
Sure, but there's no real signature. The names have become wonky with the API changes-- all we're doing is telling one service to submit some data to another service. We're not actually signing anything. How about "SubmissionResponse"?